test/jdk/sun/security/pkcs12/KeytoolOpensslInteropTest.java

```diff
@@ -23,17 +23,29 @@
 
 /*
  * @test
- * @bug 8076190 8153005
- * @library /test/lib
+ * @bug 8076190 8242151 8153005 8266182
```

Comment:
8242151 should be removed here since it was not backported.

Reply:
Thanks, removed with `a560f63cd8f`.